tnp/consent/views.py

```diff
@@ -405,33 +428,199 @@ def export_resumes(request):
     return HttpResponse('{ "message": "success" }')
 
 
 @login_required
 def user_logout(request):
     logout(request)
-    return HttpResponseRedirect('/')
\ No newline at end of file
+    return HttpResponseRedirect('/')
+
+
+# my work @abhishek981996
+
+@login_required
+def view_profile(request):
+    usercreation = User.objects.get(username=request.user)
+    personal_detail = PersonalDetail.objects.get(user=request.user)
+    education_detail = EducationDetail.objects.get(user=request.user)
+    cgpa = CGPA.objects.filter(person=education_detail)
+
+    education_detail = EducationDetail.objects.get(user=request.user)
+    cgpa = CGPA.objects.filter(person=education_detail)
+    semester_var = {}
+    for semesters in cgpa:
+        semester_var[semesters.semester] = semesters.cgpa
+
+    return render(request, 'consent/profile.html', {
+        'usercreation': usercreation,
+        'personaldetail': personal_detail,
+        'education_detail': education_detail,
+        'semester_var': semester_var,
+    })
+
+
+@login_required
+def edit_profile(request):
+    if (request.method == 'POST'):
+        personal_detail = PersonalDetail.objects.get(user=request.user)
+
+        personal_detail_form = PersonalDetailForm(
+            prefix='personal_detail_form', data=request.POST, instance=personal_detail)
+        education_detail_form = EducationDetailForm(
+            prefix='education_detail_form', data=request.POST, files=request.FILES)
+
+        print (personal_detail_form.errors.as_data())
+        print (education_detail_form.errors.as_data())
+
+        if (personal_detail_form.is_valid() and education_detail_form.is_valid()):
+            user = User.objects.get(username=request.user)
+            personal_detail = PersonalDetail.objects.get(user=request.user)
+            education_detail = EducationDetail.objects.get(user=request.user)
+            cgpa = CGPA.objects.filter(person=education_detail)
+
+            personal_detail_form = PersonalDetailForm(
+                prefix='personal_detail_form', data=request.POST, instance=personal_detail)
+            education_detail_form = EducationDetailForm(
+                prefix='education_detail_form', data=request.POST, instance=education_detail)
+
+            personal_detail = personal_detail_form.save(commit=False)
+            personal_detail.save()
+
+            education_detail = education_detail_form.save(commit=False)
+            education_detail.save()
+
+            cgpas = request.POST.getlist('cgpa')
+            sem = 1
+            for cgpa in cgpas:
+                if(cgpa):
+                    print(sem)
+                    try:
+                        cgpa_edit = CGPA.objects.get(
+                            person=education_detail, semester=sem)
+                        cgpa_edit.cgpa = cgpa
+                        cgpa_edit.save()
+                        sem += 1
+                    except Exception, e:
```

Comment:
Since you are not using the exception e variable itself, just change this line to `except: `
Also, it looks like a syntax error. 
It should be: `except Exception as e`
Your call.